sdk.d.ts

```diff
@@ -32,6 +32,7 @@ export type Aliases = {
 	aka?: string[];
 	dup?: DuplicateAlias[];
 	loc?: Record<string, string>;
+	old?: string[];
```

Comment:
The `old` alias is not implemented yet (see #5859). Can you remove it? I think that only adds confusion for type consumers.